test/test.jl

```diff
@@ -1,16 +1,28 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-using Base.Test
```

Comment:
this is necessary if you want to run this file without going through `runtests.jl`